UI/runScan.php

```diff
@@ -0,0 +1,100 @@
+<!-- 
+    When index.php is compiled and run on the terminal, it will launch this runScan.php script with messages visible.
+    Note: output messages built-into the crawler script are not visible on the server-side while running localhost:<port#> 
+    (i.e. during demonstration). However, at the end of the scan, output will be logged to the UI console available to view 
+    using: Fn F12.
+ -->
+<?php
+
+    // Allow capturing output buffer
+    ob_implicit_flush(true);
+    
+    // Locate and parse configuration file
```

Comment:
The config file only needs to be parsed when you are manually connecting to the database